Plots/Panels/NCL_panel_20.py

```diff
@@ -0,0 +1,223 @@
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc"))
+
+# Extract data from second timestep
+time_0 = ds.isel(time=0).drop_vars('time')
+time_1 = ds.isel(time=1).drop_vars('time')
+
+# Ensure longitudes range from 0 to 360 degrees
+U_0 = gvutil.xr_add_cyclic_longitudes(time_0.U, "lon")
+U_1 = gvutil.xr_add_cyclic_longitudes(time_1.U, "lon")
+
+###############################################################################
+# Create helper functions:
+
+
+def format_linegraph_axes(ax):
+    """
+    Format the axes limits, tick marks, and tick labels for the line graphs
+
+    Args:
+        ax (:class: 'matplotlib.Axes'):
+            The set of axes to be manipulated
+    """
+    # Use geocat.viz.util convenience function to set axes tick values
+    gvutil.set_axes_limits_and_ticks(ax=ax,
+                                     xlim=(-90, 90),
+                                     ylim=(-20, 50),
+                                     xticks=np.arange(-90, 91, 30),
+                                     yticks=np.arange(-20, 51, 10),
+                                     xticklabels=['90S', '60S', '30S', '0',
+                                                  '30N', '60N', '90N'])
+
+    # Use geocat.viz.util convenience function to add minor and major ticks
+    gvutil.add_major_minor_ticks(ax,
+                                 x_minor_per_major=3,
+                                 y_minor_per_major=5,
+                                 labelsize=12)
+
+
+def format_contour_axes(ax):
+    """
+    Format the axes limits, tick marks, and tick labels for the contour plots
+
+    Args:
+        ax (:class: 'matplotlib.Axes'):
+            The set of axes to be manipulated
+    """
+    # Use geocat.viz.util convenience function to set axes tick values
+    gvutil.set_axes_limits_and_ticks(ax=ax,
+                                     xlim=(-180, 180),
+                                     ylim=(-90, 90),
+                                     xticks=np.arange(-180, 181, 30),
+                                     yticks=np.arange(-90, 91, 30))
+
+    # Use geocat.viz.util convenience function to add minor and major ticks
+    gvutil.add_major_minor_ticks(ax, labelsize=8)
+
+    # Use geocat.viz.util convenience function to make plots look like NCL
+    # plots by using latitude, longitude tick labels
+    gvutil.add_lat_lon_ticklabels(ax)
+
+    # Remove the degree symbol from tick labels
+    ax.yaxis.set_major_formatter(LatitudeFormatter(degree_symbol=''))
+    ax.xaxis.set_major_formatter(LongitudeFormatter(degree_symbol=''))
+
+    # Use geocat.viz.util convenience function to set titles and labels
+    gvutil.set_titles_and_labels(ax,
+                                 maintitle='300mb',
+                                 maintitlefontsize=8)
+
+
+###############################################################################
+# Plot:
+fig = plt.figure(figsize=(12, 10))
```

Comment:
```suggestion

fig = plt.figure(figsize=(12, 10))
```
Add extra line break